src/pal/src/pal_load_file.c

```diff
@@ -22,3 +22,16 @@ char *pal_load_file_to_string(const char *filename) {
     fclose(fp);
     return file_buffer;
 }
+
+int pal_load_file_size(const char *filename) {
```

Comment:
`pal_load_file_to_string` and `pal_load_file_size` are merged into one function implementation

Reply:
Done